monitor.py

```diff
@@ -1110,6 +1117,7 @@ def parse_entry(self, line: bytes) -> MutableMapping[str, Any]:  # noqa: C901, C
                     'edmcName':  self.suit_sane_name(loc_name),
                     'id':        None,  # Is this an FDev ID for suit type ?
                     'suitId':    entry['SuitID'],
+                    'mods':      entry['SuitMods'],  # Suits can (rarely) be bought with modules installed
```

Comment:
'slots' *is* 'mods', isn't it ?

Because initially the data was only available in the CAPI `/profile` return I went with what it calls things.  See e.g. the code for `loadoutequipmodule`.

Reply:
So, we're moving towards using the Journal 'mods' for this.

But need to be sure the CAPI imported data isn't an issue.  When I checked this sort of 'slots' is definitely used on a weapon for things like Optics (the default one on TK Aphelion), but 'modifications' (with the naff 'NYI' content) seems to be what's used for installable mods.

So, we'll still have a slighty mapping issue.

Reply:
whoops didnt mean to resolve that. This is whats in the code now, no plugin devs have screamed as of yet about it.

Reply:
Discord discussion has lead me to conclude that `'mods'` is the correct name for this.  The CAPI data is a mess, follow Journal.